src/odemis/acq/fastem.py

```diff
@@ -77,6 +80,16 @@ def acquire(roa, path):
     return f
 
 
+def _configure_multibeam_hw(scanner):
+    md = scanner.getMetadata()
+    if model.MD_MULTI_BEAM_ROTATION in md:
+        scanner.rotation.value = md[model.MD_MULTI_BEAM_ROTATION]
+    else:
+        scanner.rotation.value = MULTI_BEAM_ROTATION_DEFAULT
+        logging.warning("Scanner doesn't have MULTI_BEAM_ROTATION metadata, using %s rad.",
+                        scanner.rotation.value)
+
```

Comment:
I think that to display properly the thumbnails coming from the ASM, we should add a matching MD_ROTATION. This would either be done by the mdupdater (eg, the e-beam affects the multibeam, and so the rotation VA is copied into the MD_ROTATION of the multibeam), or just explicitly here: `multibeam.updateMetadata({MD_ROTATION: scanner.rotation.value}).`

Reply:
How do we access the multibeam here? I think it's not in the stream, so it would have to be an additional argument. I think it might be better to do it in the mdupdater, what do you think?

Reply:
Yes, that sounds like a good idea. The minimum requirement for the mdupdater is that there is explicitly on the "e-beam" an affects -> multibeam. I don't think that's currently the case, but it would make sense to have it anyway.

Then, it's just a matter of adding this special case in the mdupdater class.